index.html

```diff
@@ -65,26 +65,45 @@ <h1>Hello, world!</h1>
       <!-- Example row of columns -->
       <div class="row">
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <h2>April</h2>
+          <p>Ah, who is this with twinkling feet,
+            With glad, young eyes and laughter sweet,
```

Comment:
I think we may want to add "<br>" to the end of each line to get the text styled like a poem....